packages/pn-landing-webapp/src/layout/LandingLayout.tsx

```diff
@@ -53,45 +44,45 @@ const LandingLayout = ({ children }: Props) => {
                 fontWeight: "bold",
               }}
               size="small"
-              aria-label={homeLink.ariaLabel}
-              href={homeLink.href}
+              aria-label={appData.common.pagoPALink.ariaLabel}
+              href={appData.common.pagoPALink.href}
               color="text"
               target="_blank"
               rel="noreferrer"
               disableRipple
               disableTouchRipple
             >
-              {homeLink.label}
+              {appData.common.pagoPALink.label}
             </ButtonNaked>
             <ButtonNaked
               size="small"
-              aria-label={getAppData().common.assistance.ariaLabel}
-              href={getAppData().common.assistance.href}
+              aria-label={appData.common.assistance.ariaLabel}
+              href={appData.common.assistance.href}
               color="text"
               target="_blank"
               rel="noreferrer"
               disableRipple
               disableTouchRipple
               startIcon={<HelpOutlineOutlinedIcon fontSize="inherit" />}
             >
-              {getAppData().common.assistance.label}
+              {appData.common.assistance.label}
             </ButtonNaked>
           </Stack>
         </Stack>
         <NavigationBar />
-        <Alert severity="info">{getAppData().common.alert}</Alert>
+        <Alert severity="info">{appData.common.alert}</Alert>
         <Box sx={{ flexGrow: 1 }} component="main">
           {children}
         </Box>
         <Footer
           loggedUser={false}
           companyLink={{
-            ...pagoPALink,
-            onClick: () => window.open(pagoPALink.href, "_blank"),
+            ...appData.common.pagoPALink,
+            onClick: () => window.open(appData.common.pagoPALink.href, "_blank"),
           }}
-          legalInfo={companyLegalInfo}
-          postLoginLinks={postLoginLinks}
-          preLoginLinks={preLoginLinks()}
+          legalInfo={appData.common.companyLegalInfo}
+          postLoginLinks={appData.common.postLoginLinks}
+          preLoginLinks={appData.common.preLoginLinks}
```

Comment:
I guess that the "Prodotti e servizi" list of links can translated as well by providing a URL analogous to https://selfcare.pagopa.it/assets/products.json through the `productsJsonUrl` prop which is optional to `@pagopa/mui-italia/Footer` component. Unfortunately the very title "Prodotti e servizi" is fixed in the mui-italia code, some provision should be added for that.

Reply:
On this point I think the best approach would be to change the Footer component to allow, as already possible for any other link, these data to be passed in from the client code and then open a PR on mui-italia.
I wonder why they chose this solution, but perhaps there's a valid reason I'm missing about all this thing.

Reply:
Following our discussion during today's Standup we need to better understand how to tackle this issue. Anyway I'm going to create a separate task on Jira.